File: Java/158.java
import java.util.Scanner;

/**
 *
 * 158 - Los Saltos de Mario - Golgotha's Melancholy - https://github.com/Golgotha-sMelancholy
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int prueba = sc.nextInt();
        for (int i = 1; i <= prueba; i++) {
            int muros = sc.nextInt();
            int arriba = 0;
            int abajo = 0;
            int saltos = 0;
            int anterior = 0;
            for (int j = 1; j <= muros; j++) {
                anterior = saltos;
                saltos = sc.nextInt();

                if (j >= 2 && anterior < saltos) {
                    arriba++;
                } else if (anterior > saltos) {
                    abajo++;
                } else {

                }
            }
            System.out.println(arriba + " " + abajo);
            anterior = 0;
            saltos = 0;
        }
    }

}

File: C++/158.cpp
#include <iostream>

using namespace std;

// 158 - Los Saltos de Mario - Iván - https://github.com/wildfireOfMine

int main() {
    int loop = 0;
    cin >> loop;
    for (int i = 0; i<loop; i++) {
        int subloop = 0;
        cin >> subloop;
        int input = 0;
        int aux = input;
        int high = 0;
        int low = 0;
        for (int j = 0; j<subloop; j++) {
            cin >> input;
            if (j >= 1) {
                if (input > aux) {
                    high++;
                } else if (input < aux) {
                low++;
                }
            }
            aux = input;
        }
        cout << high << " " << low << endl;
    }
    return 0;
}
